.circleci/config.yml

```diff
@@ -81,11 +81,13 @@ jobs:
             - run: pip install .
             - run: |
                   mkdir test_preparation
-                  echo "tests" > test_preparation/test_list.txt
-                  echo "tests" > test_preparation/examples_test_list.txt
-            - run: python utils/tests_fetcher.py --filter_tests
-            - run: mv test_list.txt test_preparation/filtered_test_list.txt
-            - run: mv test_repo_utils.txt test_preparation/test_repo_utils.txt
+                  echo -n "tests" > test_preparation/test_list.txt
```

Comment:
without `-n`, The command `echo` will write a `\n` to the file, which makes the condition `test_files == ["tests"]` False in `filter_tests`, and eventually failed at `f.split(os.path.sep)[1]` with index error.